components/win-users/tests/account.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate habitat_win_users;
+extern crate winapi;
+
+use std::env;
+
+use winapi::winnt::{SidTypeUser, SidTypeWellKnownGroup};
+
+use habitat_win_users::account::Account;
+
+#[test]
```

Comment:
These tests look like unit tests, I'd place them in a sub module named `tests` inside the Account module